Web-App/src/main/resources/migrations/V1.0.3__create_violations.sql

```diff
@@ -3,8 +3,8 @@ CREATE TABLE `violation` (
   `user_id` int(11) unsigned NOT NULL,
   `description` text,
   `date` date NOT NULL,
+  `modification_date` date DEFAULT NULL,
   `title` varchar(255) NOT NULL DEFAULT '',
-  `severity_rate` int(4) unsigned NOT NULL,
```

Comment:
This change should be in another migration in which severity_date column is deleted and modficationDate added.

ALTER TABLE `violation` DELETE COLUMN severity_rate;
ALTER TABLE `violation` ADD COLUMN modification_date DATETIME DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP